docs/mobile/exercises/README.md

```diff
@@ -0,0 +1,294 @@
+# Mobile exercises
+
+Writing a DataCamp for Mobile is quite different than writing a DataCamp
```

Comment:
This comment only seems appropriate for instructors who've already create a desktop course. I wouldn't lead with this comment.

Reply:
Fair point